templates/web/components/team_nav.html

```diff
@@ -9,50 +9,90 @@
         {% translate "Chatbots" %}
       </a>
     </li>
+    <li>
+      <a href="{% url 'experiments:source_material_home' request.team.slug %}"
+         {% if active_tab == 'source_material' %}class="menu-active"{% endif %}>
+        <i class="fa fa-book fa-fw"></i>
+        {% translate "Source Material" %}
+      </a>
+    </li>
+    <li>
+      <a href="{% url 'experiments:survey_home' request.team.slug %}"
+         {% if active_tab == 'survey' %}class="menu-active"{% endif %}>
+        <i class="fa-solid fa-file-circle-question fa-fw"></i>
+        {% translate "Surveys" %}
+      </a>
+    </li>
+    <li>
+      <a href="{% url 'experiments:consent_home' request.team.slug %}"
+         {% if active_tab == 'consent_forms' %}class="menu-active"{% endif %}>
+        <i class="fa-solid fa-file-contract fa-fw"></i>
+        {% translate "Consent Forms" %}
+      </a>
+    </li>
+    {% if perms.annotations.add_tag %}
+      <li>
+        <a href="{% url 'annotations:tag_home' request.team.slug %}"
+           {% if active_tab == 'tags' %}class="menu-active"{% endif %}>
+          <i class="fa fa-solid fa-tags"></i>
+          {% translate "Manage Tags" %}
+        </a>
+      </li>
+    {% endif %}
   {% endflag %}
   <li>
     <a href="{% url 'experiments:experiments_home' request.team.slug %}"{% if active_tab == 'experiments' %}class="menu-active"{% endif %}>
       <i class="fa fa-flask fa-fw"></i>
       {% translate "Experiments" %}
     </a>
     <ul>
-      <li>
-        <a href="{% url 'experiments:safety_home' request.team.slug %}"
-           {% if active_tab == 'safety_layers' %}class="menu-active"{% endif %}>
-          <i class="fa fa-shield fa-fw"></i>
-          {% translate "Safety Layers" %}
-        </a>
-      </li>
-      <li>
-        <a href="{% url 'experiments:source_material_home' request.team.slug %}"
-           {% if active_tab == 'source_material' %}class="menu-active"{% endif %}>
-          <i class="fa fa-book fa-fw"></i>
-          {% translate "Source Material" %}
-        </a>
-      </li>
-      <li>
-        <a href="{% url 'experiments:survey_home' request.team.slug %}"
-           {% if active_tab == 'survey' %}class="menu-active"{% endif %}>
-          <i class="fa-solid fa-file-circle-question fa-fw"></i>
-          {% translate "Surveys" %}
-        </a>
-      </li>
-      <li>
-        <a href="{% url 'experiments:consent_home' request.team.slug %}"
-           {% if active_tab == 'consent_forms' %}class="menu-active"{% endif %}>
-          <i class="fa-solid fa-file-contract fa-fw"></i>
-          {% translate "Consent Forms" %}
-        </a>
-      </li>
-      {% if perms.annotations.add_tag %}
+      {% flag "flag_chatbots" or %}
```

Comment:
```suggestion
      {% flag "flag_chatbots" %}
```

Reply:
The desired behaviour is to show safety layers under experiments all the time, but the rest (source material etc) should show under experiments only when the chatbots flag is disabled, otherwise with it enabled, those should show under the chatbots heading?

In which case, it seems to me to work as expected without that "or " clause. Am I missing something?